locale/update-i18n

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+FOREMAN_DIR="../../foreman"
+BASTION_DIR="../engines/bastion_katello"
+
+#this actuall pulls the strings from transifex for both directions
+make -C locale tx-update
+
+#Extract the main strings
+pushd $FOREMAN_DIR
+bundle exec rake plugin:gettext[katello]
+popd
+
+# Now extract the bastion files
+pushd $BASTION_DIR
+bundle exec grunt i18n:extract
+bundle exec grunt i18n:compile
```

Comment:
Does this work? grunt is a JS based command line so bundle exec doesn't provide any value. I guess the bundle exec does not hurt but can be dropped for clarity.

Reply:
I agree, let's remove `bundle exec` here.